Redesign Roadmap Section for Enhanced Visual Appeal and Clarity

rishyym0927 opened the pull request:
Resolve #1361 

**Description:**  
This PR addresses the need for a more structured and visually engaging roadmap section on the About page. The previous layout was simple and lacked clear organization, making it difficult to convey the project's milestones and upcoming features effectively.

### ✨ What’s New
- Redesigned the roadmap using a timeline layout for better structure.
- Incorporated icons and color coding to visually distinguish between phases.

### 🧩 Motivation
The updated design aims to:
- Enhance user experience by making the roadmap easier to read and interpret.
- Provide clear insights into the project’s direction and progress.

### 📸 Screenshots (if applicable)

https://github.com/user-attachments/assets/09a9294e-9bdd-497f-aa28-4ad9cc3f86af

arkid15r reviewed (changes requested):
Let's try it in 2x3 layout (two columns) and remove the style effects as we don't have those anywhere else.

rishyym0927 commented:
Hey @arkid15r,
I'll incorporate the two-column style as suggested. Could you please clarify what you meant by the removal of the style effect?

rishyym0927 commented:
Hey @arkid15r, I’ve removed the styling effect and switched it to a two-column layout. Check it out and let me know if anything else needs fixing—I’ll push the changes accordingly.


Uploading Recording 2025-04-14 201719.mp4…

Naveen-Pal commented on frontend/src/utils/aboutData.ts:
```diff
@@ -1,3 +1,12 @@
+  faDatabase,
+  faRobot,
+  faChartLine,
+  faCloud,
```
if arranges alphabetical it would be better

arkid15r reviewed (commented):
It seems you reverted the Roadmap changes on about page.

arkid15r reviewed (commented):
Please check my recent comment. Thank you!

arkid15r commented:
I don't see any recent progress on this, I'm going to reassign it if you're not interested.

arkid15r commented:
Pinging here as well, should we keep it open?

arkid15r commented:
Closing as abandoned.
